applications/StructuralMechanicsApplication/custom_response_functions/response_utilities/strain_energy_response_function_utility.h

```diff
@@ -269,28 +279,28 @@ class StrainEnergyResponseFunctionUtility
 				Vector perturbed_RHS = Vector(0);
 
 				// Pertubation, gradient analysis and recovery of x
-				node_i.X0() += mDelta;
+				node_i.X0() += delta;
```

Comment:
I havent checked in detail but it seems like the following lines are repeated several times

Reply:
:+1:

Reply:
A now i know what you meant, they are similar but not the same.